src/shared/prerendered-app/util.css

```diff
@@ -16,9 +16,5 @@
     font: inherit;
     padding: 0;
     margin: 0;
-
-    &:focus {
-      outline: none;
-    }
```

Comment:
I think we'd need to review the impact of this across the rest of the app before landing this.

Reply:
I understand where you’re coming from although I respectfully disagree. Right now, buttons are being stripped out of their focus outline entirely, which is a critical accessibility issue. I would argue the default browser outline is better than no outline at all. These focus states can then be incrementally updated to fit the design more.

Reply:
That being said, I’m happy removing that part of the diff if you would feel more comfortable.

Reply:
I didn't say it should be removed, I just want it to be reviewed.

Specifically, I want to avoid buttons that look "stuck" for mouse users.

Reply:
I see, that makes sense.

Reply:
> Specifically, I want to avoid buttons that look "stuck" for mouse users.

Use focus-visible polyfill, the focus outline only triggers when activated by keyboard use https://github.com/WICG/focus-visible